### PR TITLE
Further code style changes.

### PR DESCRIPTION
**Pull request by tanuck**
Here are some further updates to the `./src` and `./tests` files to bring the library closer to the PSR2 standard. I've only tested this against the PSR2 standard rules that come with PHPCS v2, so it isn't very strict, but its a start!

I've also included PHPCS as a composer dev dependency - but I'm open to suggestions as to whether this is necessary?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
     "require": {
         "php": ">=5.3.3",
-        "ext-json": "*"
+        "ext-json": "*",
+        "squizlabs/php_codesniffer": "2.x"
```
Comment (GrahamCampbell) [inline composer.json]: :-1:

Comment (GrahamCampbell) [inline composer.json]: There's no way this can go in require.

Comment (tanuck) [inline composer.json]: Yup, noticed this straight after submitting the PR :-1:

**Comment by zimbatm** [comment]
Cheers :tada: :balloon: :cake:

**Comment by tanuck** [comment]
:+1:

**Pull request merged**